std_msgs/Header header
bool detected
float32 bearing     # radians; 0 = straight ahead; +left, -right
float32 distance    # arbitrary units or meters, -1.0 if unknown